oximeter/oximeter/schema/bfd-session.toml

```diff
@@ -5,7 +5,7 @@ name = "bfd_session"
 description = "A Bidirectional Forwarding Protocol (BFD) session"
 authz_scope = "fleet"
 versions = [
-    { version = 1, fields = [ "hostname", "rack_id", "sled_id", "peer" ] },
+    { version = 1, fields = [ "hostname", "peer", "rack_id", "sled_id", "sled_model", "sled_revision", "sled_serial", "switch_id", "switch_fab", "switch_lot", "switch_wafer", "switch_wafer_loc_x", "switch_wafer_loc_y", "switch_model", "switch_revision", "switch_serial", "switch_slot" ] },
```

Comment:
@bnaecker, there's a question about which switch and sled fields we really need. Is it just `switch_id` (what Ry recommended)? Do we include anything beyond `sled_id` for sled stuff?  I laid everything out as we get all the data (either via smf or dpd-client), but it's easy to restrict this. Thoughts?